### PR TITLE
v1-series-transformer is now based on sections-transformer.

### PR DESCRIPTION
**Pull request by peteclark-ft**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -65,54 +67,45 @@ func main() {
...
-	cacheFileName := app.String(cli.StringOpt{
-		Name:   "cache-file-name",
-		Value:  "cache.db",
-		Desc:   "Cache file name",
-		EnvVar: "CACHE_FILE_NAME",
+		EnvVar: "SLICES",
 	})
 
-	tmeTaxonomyName := "topics"
+	tmeTaxonomyName := "series"
```
Comment (tosan88) [inline main.go:77]: Is this the correct name coming from TME - KnowledgeBases? I can't find it

Comment (peteclark-ft) [inline main.go:77]: Hi - no it's not; there isn't a taxonomy for series setup yet in TME (as far as I know), so this is just a placeholder.

Comment (tosan88) [inline main.go:77]: Ah I see. :)
Also note, that this should be _exactly_ what comes out from TME, e.g. _GL_ for locations

**Comment by tosan88** [comment]
`sample_sections.xml` is not used, can be deleted. 
Otherwise looks good to me

File: main.go (hunk 1 above)
Comment (peteclark-ft) [inline main.go:77]: We'll need to re-evaluate everything once we get the full spec for Series from TME; but just trying to get some scaffolding in place early. Thanks for reviewing though!

**Comment by kghatton** [comment]
:+1:

**Pull request merged**